src/sentry/integrations/vercel/integration.py

```diff
@@ -76,6 +77,85 @@ def get_organization_config(self):
 
         return fields
 
+    def update_organization_config(self, data):
+        # data = {"project_mappings": [[sentry_project_id, vercel_project_id]]}
+
+        metadata = self.model.metadata
+        vercel_client = VercelClient(metadata["access_token"], metadata.get("team_id"))
+        config = self.org_integration.config
+        sentry_project_id = data["project_mappings"][-1][0]
+        vercel_project_id = data["project_mappings"][-1][1]
+        sentry_project = Project.objects.get(id=sentry_project_id)
+        sentry_project_dsns = ProjectKey.objects.filter(project=sentry_project).all()
+        enabled_dsns = [dsn for dsn in sentry_project_dsns if dsn.is_active]
+        if enabled_dsns:  # in case there are no enabled DSNs
+            sentry_project_dsn = enabled_dsns[0].get_dsn(public=True)
+        else:
+            raise
+        try:
+            self.create_secret(
+                vercel_client, vercel_project_id, "SENTRY_ORG", sentry_project.organization.slug
+            )
+        except Exception:
+            raise
+
+        try:
+            self.create_secret(
+                vercel_client, vercel_project_id, "SENTRY_PROJECT", sentry_project.slug
+            )
+        except Exception:
```

Comment:
@ceorourke No need to have try/except like this if all we do is re-raise the exception